src/Sylius/Component/Core/Promotion/Checker/Rule/CustomerGroupRuleChecker.php

```diff
@@ -39,7 +39,11 @@ public function isEligible(PromotionSubjectInterface $subject, array $configurat
             return false;
         }
 
-        if ($configuration['groups'] === $customer->getGroup()->getId()) {
+        if (is_null($customer->getGroup())) {
```

Comment:
We have the following convention for checking for `null` value: `if (null === $customer->getGroup()) {`